PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -168,6 +171,13 @@ PATElectronProducer::PATElectronProducer(const edm::ParameterSet & iConfig) :
   //   }
   //   isoDepositTokens_ = edm::vector_transform(isoDepositLabels_, [this](std::pair<IsolationKeys,edm::InputTag> const & label){return consumes<edm::ValueMap<IsoDeposit> >(label.second);});
 
+  // for mini-iso
+  computeMiniIso_ = iConfig.getParameter<bool>("computeMiniIso");
+  miniIsoParamsE_ = iConfig.getParameter<std::vector<double> >("miniIsoParamsE");
```

Comment:
is there a check on the length of these vectors? Best to confirm they have the correct number of elements